motion-1/Overview.bs

```diff
@@ -608,7 +608,7 @@ how to process an 'offset-rotation'.
 <h3 id="offset-shorthand">Offset shorthand: The 'offset' property</h3>
 <pre class='propdef'>
 	Name: offset
-	Value: &lt;'offset-path'> && &lt;'offset-distance'> && &lt;'offset-position'> && &lt;'offset-anchor'> && &lt;'offset-rotation'>
+	Value: [ <<offset-position>> <<offset-rotation>>? <br> | <<offset-position>>? <<offset-path>> [ <<offset-distance>> || <<offset-rotation>> ]? ] <br> [ / <<offset-anchor>> ]?
```

Comment:
`[<offset-position>]? [<offset-path> [<offset-distance> || <offset-rotation>]? ]? [ / <offset-anchor> ]?`
seems good to me.

This can solve the problem:

1. The ambiguity in `<position>` value between `offset-position` and `offset-anchor`
 - `/` is always between `offset-position` and `offset-anchor`

2. The ambiguity in `auto` value among `offset-position`, `offset-rotation`, and `offset-anchor`
 - The value of `offset-path` is always between `offset-position` and `offset-rotation`
 - `/` is always between `offset-position` and `offset-anchor`
 - `/` is always between `offset-rotation` and `offset-anchor`

Also, when I think about the dependencies of offset-* properties on `offset-path`,
- offset-position: Independent
 -  It would be used without a path.
- offset-anchor: Independent
 - It would be used without a path.
- offset-distance: Definitely dependent
 - It won't work without a path.
- offset-rotation: Dependent
 - If 'auto' or 'reverse' value is given, it wouldn't work without a path.

   eg) `offset-position: center; offset-rotation: auto;`
         'auto' for offset-rotation is the angle of the direction of a path.
         'auto' cannot be computed because there isn't a defined path.

Therefore, I think `<<offset-position>> <<offset-rotation>>?` could be omitted.